randovania/games/dread/logic_database/Dairon.txt

```diff
@@ -1002,7 +1002,7 @@ Extra - asset_id: collision_camera_012
   > Ledge before Slide Jump
       All of the following:
           Morph Ball
-          Flash Shift or Use Spin Boost
+          Flash Shift or Lay Cross Bomb or Use Spin Boost
```

Comment:
Should this have a movement trick attached? For `Ledge before Slide Jump -> Door to Transport to Artaria` it makes sense trickless, but from this direction it seems like a small ledge to balance on.

Reply:
You basically need to be on the edge to utilize the full boost of the cross bomb as it is. I don't think it would need a movement trick if you have to be on the edge anyway.

Reply:
I played around with it, and decided that this can be trickless both ways on the basis that it's very hard to screw up.

Reply:
Well, if you mess it up you probably die. That's not any worse than the other direction though, I was mainly just worried about how much room you had to set it up. It seemed like it would be easy to roll too far and fall off, which doesn't feel very trickless to me.